### PR TITLE
Merge main into dev

### PR DESCRIPTION
**Pull request by pictos**
Since I had some merge conflicts I open this PR to merge the `main` branch into `develop`

### CODE REVIEW

**Comment by pictos** [comment]
I'm not sure why the CI can't build this... Locally everything works. I'll try to do a dumb commit to force the build.

```
"/Users/runner/work/1/s/src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj" (Build target) (1:54) ->
  /Library/Frameworks/Mono.framework/Versions/6.4.0/lib/mono/msbuild/Current/bin/Roslyn/Microsoft.CSharp.Core.targets(59,5): error MSB3105: The item "ObjectModel/Internals/BaseCommand.netstandard.cs" was specified more than once in the "Sources" parameter.  Duplicate items are not supported by the "Sources" parameter. [/Users/runner/work/1/s/src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj]

```

**Comment by pictos** [comment]
@jfversluis pinging you since you're our devops hero!

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj [hunk 1]
```diff
@@ -79,12 +79,18 @@
...
   </ItemGroup>
   <ItemGroup Condition=" $(TargetFramework.StartsWith('netstandard2.0')) ">
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
+    <Compile Include="**\*.netstandard.cs" />
+    <Compile Include="**\*.netstandard.*.cs" />
   </ItemGroup>
   <ItemGroup Condition=" $(TargetFramework.StartsWith('netstandard2.1')) ">
+    <Compile Include="**\*.netstandard.cs" />
+    <Compile Include="**\*.netstandard.*.cs" />
```
Comment (jfversluis) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj:93]: I think the build error comes from this. This is already added on line 69 & 70. We don't need these again?

Comment (pictos) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj]: @jfversluis if with that change the build still fails :/

**Pull request closed without merging**